web/src/pages/profile.tsx

```diff
@@ -567,18 +567,6 @@ const ProfilePage = (props: Props): ReactElement => {
           <NonEssentialQuestionsSection />
         </ProfileField>
 
-        <ProfileField
```

Comment:
Removed the field from the `starting` profile because it can no longer appear there